Spotify Clone/index.html

```diff
@@ -1,94 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-    <head>
-        <meta charset="UTF-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">

```

Comment:
update ur branch

Reply:
Sir , this is todo app edit functionality branch,
So, it is different from whatever screenshot you are sending, That is file of different branch, it got reflected in this ,then it would be extra file for this PR. That's why I deleted those files.

Reply:
@mayank-del u  deleted those files if i merge this PR those files will be deleted from my repo

Reply:
restore those files